_includes/Analytics.html

```diff
@@ -0,0 +1,29 @@
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-97923818-10"></script>
+<script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+    gtag('config', 'UA-97923818-10');
+</script>
+<script>
+    (function () {
+      var didInit = false;
+      function initMunchkin() {
+      if (didInit === false) {
+      didInit = true;
+      Munchkin.init('353-SAE-487');
+      }
+      }
+      var s = document.createElement('script');
+      s.type = 'text/javascript';
+      s.async = true;
+      s.src = '//munchkin.marketo.net/munchkin.js';
+      s.onreadystatechange = function () {
+      if (this.readyState == 'complete' || this.readyState == 'loaded') {
+      initMunchkin();
+      }
+      };
+      s.onload = initMunchkin;
+      document.getElementsByTagName('head')[0].appendChild(s);
+      })();
+</script>
```

Comment:
@jlgonzalez099 I don't see where this file is used. Is it supposed to be?

Reply:
@mmorey  Sorry. Thought I had removed this file. But I see it has been done. Thanks